src/server/server.ts

```diff
@@ -274,8 +274,10 @@ async function serveEditionsArticlePost(
 		// The "req.body" should contain a 'Content' object which fetched by the
 		// Edition backend from the capi
 		const content = await capiContentDecoder(req.body);
+		const footballContent = await getFootballContent(content);
```

Comment:
This will make a call to MAPI to get the football data from within Apps-Rendering. I think instead you want the Editions backend to POST the football data through to AR as part of the `RenderingRequest`?

Reply:
As discussed IRL we (myself and Jamie) think for now it will make more sense to do this in Apps-Rendering and eventually migrate the functionality to Editions Backend, once it's in a place where its easy to do so ie after various updates which are currently blockers.

@frankie297 do you have any objections to this approach?